app/src/test/java/gov/va/vro/controller/VroControllerTest.java

```diff
@@ -201,21 +201,6 @@ void fullHealthAssessmentMalformedJson() {
     assertEquals(HttpStatus.BAD_REQUEST, responseEntity.getStatusCode());
   }
 
-  @Test
-  void serverResponseUnsupportedHttpMethod() {
-    Map<String, String> headers = new HashMap<>();
-    headers.put("accept", "application/json");
-    headers.put("content-type", "application/json");
-    String url = "/v1/fetch-claims";
-    String sampleRequestBody = "{ \"one\":\"one\", \"two\":\"two\",}";
-    var getResponseEntity = get(url, headers, ClaimProcessingError.class);
-    var postResponseEntity = post(url, sampleRequestBody, headers, ClaimProcessingError.class);
-    var putResponseEntity = put(url, sampleRequestBody, headers, ClaimProcessingError.class);
-    assertEquals(HttpStatus.OK, getResponseEntity.getStatusCode());
-    assertEquals(HttpStatus.METHOD_NOT_ALLOWED, postResponseEntity.getStatusCode());
-    assertEquals(HttpStatus.METHOD_NOT_ALLOWED, putResponseEntity.getStatusCode());
-  }
-
```

Comment:
This was added to test the fix from sending 500 Internal Server Error to 405 Unsupported Method -- I didn't realize we were removing `/fetch-claims` -- so rather than removing this, can we just update the URL to something else like `/full-health-data-assessment`?

Reply:
yep, no problem!